docs/t-sql/queries/from-using-pivot-and-unpivot.md

```diff
@@ -38,22 +38,43 @@ The syntax for `PIVOT` is simpler and more readable than the syntax that may oth
 The following syntax summarizes how to use the `PIVOT` operator.  
   
 ```syntaxsql  
-SELECT <non-pivoted column>,  
-    [first pivoted column] AS <column name>,  
-    [second pivoted column] AS <column name>,  
-    ...  
-    [last pivoted column] AS <column name>  
+SELECT [<non-pivoted column> AS <column name>,]
+    ...
+    <first pivoted column> AS <column name>,
+    <second pivoted column> AS <column name>,
+    ...
+    <last pivoted column> AS <column name>
 FROM  
     (<SELECT query that produces the data>)   
     AS <alias for the source query>  
 PIVOT  
 (  
     <aggregation function>(<column being aggregated>)  
 FOR   
-[<column that contains the values that will become column headers>]   
+<column that contains the values that will become column headers>
     IN ( [first pivoted column], [second pivoted column],  
     ... [last pivoted column])  
-) AS <alias for the pivot table>  
+) AS <alias for the pivot table>
+<optional ORDER BY clause>;  
+```  
+The following syntax summarizes how to use the `UNPIVOT` operator.  
+  
+```syntaxsql  
+SELECT <non-pivoted column> AS <column name>,
+    ...
+    <output column for names of the pivot columns>,
+    <new output column created for values in result of the source query>
+FROM  
+    (<SELECT query that produces the data>)   
+    AS <alias for the source query>  
+UNPIVOT
+(  
+    <new output column created for values in result of the source query>
+FOR   
+<output column for names of the pivot columns>
+    IN ( [first pivoted column], [second pivoted column],  
+    ... [last pivoted column])  
+)
```

Comment:
This syntax probably better suited in the [UNPIVOT Example](https://learn.microsoft.com/en-us/sql/t-sql/queries/from-using-pivot-and-unpivot?view=sql-server-ver16#unpivot-example) section.

Reply:
The Remarks directly below the Syntax section directly reference the UNPIVOT clause. So if the Remarks are there, then I would say the UNPIVOT should remain above those Remarks.

The other option is:
- Change "Syntax" to "PIVOT Syntax"
- Add a "UNPIVOT Syntax" Section above the "UNPIVOT Example"
- Move the first paragraph of the Remarks, concerning the UNPIVOT syntax, below the "UNPIVOT Syntax" section.

Reply:
I think it's probably best if we add some more context before the syntax to include `UNPIVOT`. Thanks for the suggestions!